met-web/config/env.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const fs = require('node:fs');
+const path = require('node:path');
+const paths = require('./paths');
+
+// Make sure that including paths.js after env.js will read .env variables.
+delete require.cache[require.resolve('./paths')];
+
+const NODE_ENV = process.env.NODE_ENV;
+if (!NODE_ENV) {
+  throw new Error(
+    'The NODE_ENV environment variable is required but was not specified.'
+  );
+}
+
+// https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
+const dotenvFiles = [
+  `${paths.dotenv}.${NODE_ENV}.local`,
+  // Don't include `.env.local` for `test` environment
+  // since normally you expect tests to produce the same
+  // results for everyone
+  NODE_ENV !== 'test' && `${paths.dotenv}.local`,
+  `${paths.dotenv}.${NODE_ENV}`,
+  paths.dotenv,
+].filter(Boolean);
+
+// Load environment variables from .env* files. Suppress warnings using silent
+// if this file is missing. dotenv will never modify any environment variables
+// that have already been set. Variable expansion is supported in .env files.
+// https://github.com/motdotla/dotenv
+// https://github.com/motdotla/dotenv-expand
+const dotenvExpand = require('dotenv-expand');
+dotenvFiles.forEach(dotenvFile => {
+  if (fs.existsSync(dotenvFile)) {
+    const envResult = require('dotenv').config({
+      path: dotenvFile,
+      quiet: true,
+    });
+
+    dotenvExpand.expand(envResult);
+  }
+});
+
+// We support resolving modules according to `NODE_PATH`.
```

Comment:
After looking this up, it looks like node_path isn't deprecated, but it's being abandoned in favour of tsconfig.json compilerOptions.paths.@components. I'm not sure if that's relevant in your situation though.